manifest.json

```diff
@@ -62,22 +52,24 @@
   "commands": {
     "fill-form": {
       "suggested_key": {
-        "default": "MacCtrl+Shift+F"
+        "default": "Ctrl+Shift+F",
```

Comment:
What's the reason behind this shift? On chrome macctrl doesn't mean ctrl or cmd anymore?

Reply:
On Firefox, `MacCtrl` has the sam meaning, but chrome only supports providing the shortcuts separately.